.cicd/eosio-contracts-unittest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+[[ ! -z "$CONTRACTS_VERSION" ]] || export CONTRACTS_VERSION="$(cat "$PIPELINE_CONFIG" | jq -r '.dependencies["eosio.contracts"]')"
+git clone -b "$CONTRACTS_VERSION" https://github.com/EOSIO/eosio.contracts.git 
+
+# Docker Commands
+ARGS=${ARGS:-"--rm --init -v $(pwd):$MOUNTED_DIR"}
+. $HELPERS_DIR/docker-hash.sh
+
+if [[ $BUILDKITE == true ]]; then
+    # Generate Base Images
+    $CICD_DIR/generate-base-images.sh
+    FULL_TAG='eosio/ci-contracts-builder:base-ubuntu-18.04-develop'
+    TEST_CONTRACTS_COMMAND="cd $MOUNTED_DIR/build_eosio_contracts/tests && ctest -j $JOBS --output-on-failure -T Test"
+    
+    # Load BUILDKITE Environment Variables for use in docker run
+    if [[ -f $BUILDKITE_ENV_FILE ]]; then
+        evars=""
+        while read -r var; do
+            evars="$evars --env ${var%%=*}"
+        done < "$BUILDKITE_ENV_FILE"
+    fi
+    
+    echo '--- :arrow_up: Sanity tests for eosio.contract wasm and abi files'
+    eval docker run $ARGS $evars $FULL_TAG bash -c \"$TEST_CONTRACTS_COMMAND\"
```

Comment:
Here, again, we should print significant commands before running them.
```bash
DOCKER_RUN="docker run $ARGS $evars $FULL_TAG bash -c \"$TEST_CONTRACTS_COMMAND\""
echo "$ $DOCKER_RUN"
eval $DOCKER_RUN
```
This way, the customer can see what the CI system is trying to do and replicate it locally.

Reply:
Done.